src/dodal/devices/areadetector/plugins/MJPG.py

```diff
@@ -70,7 +70,7 @@ def get_snapshot():
             try:
                 response = requests.get(url_str, stream=True)
                 response.raise_for_status()
-                with Image.open(response.raw) as image:
+                with Image.open(response.content) as image:
```

Comment:
@DominicOram this is the change I am not 100% on.

Reply:
As discussed, this is failing on main too, I will fix in a new PR

Reply:
https://github.com/DiamondLightSource/dodal/pull/777